charts/ks-devops/values.yaml

```diff
@@ -88,7 +88,7 @@ affinity: {}
 
 charts:
   jenkins:
-    enabled: true
+    enabled: false
```

Comment:
It should be true. Because Jenkins is still the default option.

Reply:
Okay, but if I set it as true, when installing tekton, we are supposed to set it to false.

Reply:
Users can provide two flags for the installation. Set `jenkins` is false, and `tekton` is true. This is not the perfect solution. But we still need to have a default option.